packages/@uppy/aws-s3-multipart/src/index.js

```diff
@@ -206,6 +265,9 @@ class HTTPCommunicationQueue {
 
   async resumeUploadFile (file, chunks, signal) {
     throwIfAborted(signal)
+    if (chunks.length === 1) {
```

Comment:
Should this be checking shouldUseMultipart like we do for uploadFile?

https://github.com/transloadit/uppy/pull/4205/files#diff-bfb35efb13daef9d39e5d9a560b0a4ac5b0ad1c4c76bb461dc3724d5e5d7da7cR249-R252